src/factom-event-emitter.js

```diff
@@ -43,9 +48,22 @@ Object.freeze(FACTOM_EVENT);
  * emitter.on('FA29eyMVJaZ2tbGqJ3M49gANaXMXCjgfKcJGe5mx8p4iQFCvFDAC', (transaction) => ...);
  * // Listen to any new entries in a given chain
  * emitter.on('4060c0192a421ca121ffff935889ef55a64574a6ef0e69b2b4f8a0ab919b2ca4', (entryBlock) => ...);
+ * // Listen to any pending transactions involving a given Factoid address
+ * emitter.on(FactomEventEmitter.getSubscriptionToken({
+ *  eventType: 'newPendingTransaction', address: 'FA29eyMVJaZ2tbGqJ3M49gANaXMXCjgfKcJGe5mx8p4iQFCvFDAC'
+ * }), (pendingTransaction) => ...);
  */
 class FactomEventEmitter extends EventEmitter {
-    constructor(cli, opts) {
+    /**
+     * Given an event configuration object returns a tokenized string
+     * @param {object} event
+     * @returns {string}
+     */
+    static getSubscriptionToken(event) {
+        return `${event.eventType}:${event.address}`;
+    }
```

Comment:
Ideally we should be able to reuse that method for pending entries too, so that would be nice to make it agnostic to 'address'. I am thinking `topic` could be good generic name that could be populated with an address for new pending transactions and with a chainId for new pending entries. What do you think?